ocaml/xapi/xapi_gpumon.mli

```diff
@@ -15,3 +15,18 @@
 (** Stop gpumon if it's running, perform f, then start gpumon if
   * no other threads which require gpumon to be stopped are running. *)
 val with_gpumon_stopped : ?timeout:float -> (unit -> 'a) -> 'a
+
+module Nvidia : sig
+  (** The key used in the metadata assoc-list *)
+  val key: string
+
+  (** Fetch metadata about the PGPU from the driver, and return
+   *  [(key, metadata)] where key is a fixed value and metadata
+   *  is the opaque string of data from the graphics driver.
+   *  IMPORTANT: This must be called on the host that has the GPU installed in it. *)
+  val get_pgpu_compatibility_metadata:
+    __context:Context.t ->
+    pgpu:[ `PGPU ] API.Ref.t ->
+    (string * string Gpumon_client.Client.RPCM.t) list
```

Comment:
What is this `Gpumon_client.Client.RPCM.t`? I'd expect just a `(string * string) list` here. The caller of this function shouldn't care about Gpumon client types.